File: app/src/main/java/com/loony/timelapsemaker/camera/OnTimelapseStateChangeListener.java
package com.loony.timelapsemaker.camera;

/**
 * Created by Kamil on 7/21/2017.
 */

public interface OnTimelapseStateChangeListener {
    void onProgress(int capturedPhotos, byte[] imageCaptured);
    void onComplete();
    void onFail();
}

File: app/src/main/java/pl/kflorczyk/timelapsemaker/timelapse/OnTimelapseStateChangeListener.kt
package pl.kflorczyk.timelapsemaker.timelapse

/**
 * Created by Kamil on 2017-12-11.
 */
interface OnTimelapseStateChangeListener {
    fun onInit()
    fun onCapture(bytes: ByteArray?)
    fun onFail(msg: String)
}